cfme/tests/networks/provider/test_events.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+"""This module tests events that are invoked by Network entities (e.g. subnets)."""
+import pytest
+import yaml
+
+from cfme.networks.provider.nuage import NuageProvider, NetworkProvider
+
+
+pytestmark = [
+    pytest.mark.provider([NetworkProvider], scope="module")
+]
+
+
+@pytest.fixture(scope="function")
+def setup_remote_entities(provider):
+    provider.setup_test_entities_on_provider()
+
+
+@pytest.fixture(scope="function")
+def teardown_remote_entities(provider):
+    provider.teardown_test_entities_on_provider()
```

Comment:
**REQUIRED**
If it's supposed to be executed in the teardown then it should like this:
```python
@pytest.yield_fixture
def teardown_remote_entities(provider):
    yield
    provider.teardown_test_entities_on_provider()
```
But it doesn't make sense for me to divide these fixtures in to two separate functions. Create just one:
```python
@pytest.yield_fixture
def prerequisites():
    provider.setup_test_entities_on_provider()
    yield
    provider.teardown_test_entities_on_provider()
```